tests/test_models.py

```diff
@@ -1077,6 +1077,43 @@ def test_annotate_eqtransformer(parallelism):
     )  # Ensures classify succeeds even though labels are unknown
 
 
+@pytest.mark.parametrize(
```

Comment:
Nitpicking, but it would be more beautiful to also parametrise the model choice here.

Reply:
Done.